### PR TITLE
Improvement/change home view

### PR DESCRIPTION
**Pull request by veymarjoel**

### CODE REVIEW

File: src/containers/Home.js [hunk 1]
```diff
@@ -8,8 +8,10 @@ import { getQueryParams } from '../utils/utils';
 import Loader from '../components/Loader/Loader';
 // Actions
 import { search, clear } from '../actions/listUsers.actions';
-import CustomList from '../components/CustomList/CustomList';
-import Card from '../components/Card/Card';
+import CardCollection from '../components/CardList/CardCollection';
+// import CustomList from '../components/CustomList/CustomList';
```
Comment (cinthia-neko) [inline src/containers/Home.js]: we should remove this.

Comment (veymarjoel) [reply src/containers/Home.js]: done thanks

File: src/components/CardList/CardCollection.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.collection-list {
```
[the remark this answers is not in the document]
Comment (veymarjoel) [reply src/components/CardList/CardCollection.css]: done

File: src/components/CardList/CardCollection.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.item-margin {
```
[the remark this answers is not in the document]
Comment (veymarjoel) [reply src/components/CardList/CardCollection.css]: done

File: src/components/CustomList/CustomList.css [hunk 4]
```diff
@@ -7,4 +7,13 @@
   border: 2px solid #D4D4D5;
   padding: 12px;
 }
-  
\ No newline at end of file
+
+.stack-list {
```
[the remark this answers is not in the document]
Comment (veymarjoel) [reply src/components/CustomList/CustomList.css]: corrected

File: src/components/CustomList/CustomList.css [hunk 5]
```diff
@@ -7,4 +7,13 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.item-margin {
```
[the remark this answers is not in the document]
Comment (veymarjoel) [reply src/components/CustomList/CustomList.css]: corrected

**Pull request merged**